installers/flash-tarball/radiasoft-download.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# To run: curl radia.run | bash -s flash-tarball
+#
+set -euo pipefail
+
+flash_main() {
+    local d="$PWD"
+    if [[ ! -f "$d/FLASH-4.6.2.tar.gz" ]]; then
+        install_err "$d/FLASH-4.6.2.tar.gz must exist"
+    fi
+    cd ~/src/radiasoft/rsflash
+    git checkout --quiet master
+    git clean --quiet --force -d
+    git pull --quiet
+    local r=()
+    for x in \
+        "CapLaser3D 47a641aa467ff48c1337a69e6c3a6778e5b854ae" \
+        "CapLaserBELLA master" \
+    ; do
+        local x=( $x )
+        git checkout --quiet "${x[1]}"
+        cd "config/${x[0]}"
+        # POSIT: Matches sirepo.sim_data._flash_problem_files_archive_basename
+        n="problemFiles-archive.${x[0]}.zip"
+        zip --quiet "$d/$n" {*F90,Config,Makefile}
+        r+=($n)
+        cd ../..
+    done
+    cd "$d"
+    r+=("$(flash_patch_makefile)")
+    tar czf flash.tar.gz "${r[@]}"
+    rm -rf "${r[@]}"
+    # TODO(e-carlin):  what about outside of dev do we put it in $p?
+    local p="$HOME/src/radiasoft/rsconf/proprietary/"
```

Comment:
@robnagler is this ok hardcoded?

Reply:
1d07dd5 . I now assume we are running in proprietary because `codes_download rsflash` downloads in $PWD so better to do that in proprietary.